### PR TITLE
 Added docs for say_with_time method

### PR DESCRIPTION
**Pull request by prathamesh-sonpatki**
- say method is internally called only from say_with_message so nodoc'd it.

### CODE REVIEW

**Comment by prathamesh-sonpatki** [comment]
Thanks to DocsDoctor - http://www.docsdoctor.org/doc_methods/332695

**Comment by matthewd** [comment]
That's been public API for a very long time.

**Comment by prathamesh-sonpatki** [comment]
Thanks @matthewd for the review. I wanted to add documentation for `say_with_message` but also noticed that `say` is called only from `say_with_message`, so nodo'd it. Should I remove the nodoc part and keep documentation?

**Comment by rafaelfranca** [comment]
@prathamesh-sonpatki yes please

File: activerecord/lib/active_record/migration.rb [hunk 1]
```diff
@@ -622,10 +622,12 @@ def announce(message)
       write "== %s %s" % [text, "=" * length]
     end
 
-    def say(message, subitem=false)
+    def say(message, subitem=false) # :nodoc:
       write "#{subitem ? "   ->" : "--"} #{message}"
     end
 
+    # Prints customized +message+ to the console along with benchmarks
+    # describing how long each step in migration took.
```
Comment (matthewd) [inline activerecord/lib/active_record/migration.rb]: I'm not sure this is a terribly accurate description of what this method does :confused:

Comment (prathamesh-sonpatki) [inline activerecord/lib/active_record/migration.rb]: @matthewd @rafaelfranca Please review https://github.com/prathamesh-sonpatki/rails/commit/fcddac9b9e36766eb58c144bef0cbd599a109526. Can we open this PR?

**Pull request closed without merging**